src/Event/StartProductSync.php

```diff
@@ -41,6 +41,13 @@ function() {
 				$this->on_settings_sync();
 			}
 		);
+
+		add_action(
+			'woocommerce_gla_rules_change',
```

Comment:
Since this is a global action do we want to clarify it's a mapping rules change?

```suggestion
			'woocommerce_gla_mapping_rules_change',
```